File: Leetcode/1252. Cells with Odd Values in a Matrix/solution.py
class Solution:
    def oddCells(self, n: int, m: int, indices: List[List[int]]) -> int:
        # Count the indices frequency in row an columns
        rows = [0] * n
        cols = [0] * m

        for row, col in indices:
            rows[row] ^= 1
            cols[col] ^= 1

        # Count odd numbers
        odds = 0
        for i in range(n):
            for j in range(m):
                if rows[i] ^ cols[j] == 1:
                    odds += 1
        return odds

File: Leetcode/1252. Cells with Odd Values in a Matrix/solution.go
func oddCells(n int, m int, indices [][]int) int {
	rows := make([]int, n)
	cols := make([]int, m)
	odds := 0

	for _, pos := range indices {
		rows[pos[0]] += 1
		cols[pos[1]] += 1
	}

	for i := 0; i < n; i++ {
		for j := 0; j < m; j++ {
			if (rows[i]+cols[j])%2 == 1 {
				odds += 1
			}
		}
	}

	return odds
}